doc/03_configuration.md

```diff
@@ -94,6 +94,24 @@ The primary advantage to using the `#meta-merge` tag over aero's
 `#merge` is that meta-merge will do a deep merge of the tree; meaning
 developers can more easily inherit changes to the base configuration.
 
+Meta-merge is also applied by default for ???. You can use `^:prepend`, :`^append`, and` ^:replace` to
```

Comment:
Still a `???` that I think needs filling in. Apart from that it looks good.

Reply:
I switched to a computer that didn't have the latest changes, rebased, and then force-pushed, meaning the changes were lost from GitHub. I restored them from another copy.